config/samples/hive_v1alpha1_clusterimageset.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: hive.openshift.io/v1alpha1
+kind: ClusterImageSet
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: clusterimageset-sample
+spec:
+  hiveImage: registry.svc.ci.openshift.org/openshift/hive-v4.0:hive
+  releaseImage: quay.io/openshift-release-dev/ocp-release:4.0.0-0.6
```

Comment:
This is probably going to be the primary use case to install a cluster going forward, we should probably factor it into the cluster-deployment.yaml template and e2e tests. We could setup make deploy to automatically setup an openshift-latest that points to latest CI images (assuming there is one for release that we can pull)? Or should we drop it into the clusterdeployment template as another object?

Reply:
We can just add it in as another object in the clusterdeployment template. If ok with you, I'll handle in a follow-up PR.

Reply:
Sounds good.